src/components/map/Map.js

```diff
@@ -81,7 +88,7 @@ class Map extends Component {
       <GoogleMap
         defaultZoom={defaultZoom}
         center={this.state.center}
-        onClick={this.onMapClick}
+        onClick={clickEnabled ? this.onMapClick : (() => {})}
```

Comment:
You can just pass null instead of an empty function.
I think you can also express it like `clickEnabbled && this.onMapClick`

Reply:
If I do so I get the following error `Failed prop type: Invalid prop `onClick` of type `boolean` supplied to `GoogleMap`, expected `function`.` since the onClick is a prop for the `GoogleMap` component.

Reply:
Setting the default as `null` works though